test/test-lib-nodejs.sh

```diff
@@ -528,19 +528,59 @@ function test_safe_logging() {
   fi
 }
 
+
+function ct_pull_or_import_postgresql() {
+  postgresql_image="quay.io/sclorg/postgresql-12-c8s"
+  image_short="postgresql:12"
```

Comment:
We could already use v15.

Reply:
This needs to be tested. Deeply.

Reply:
Fixed by https://github.com/sclorg/s2i-nodejs-container/pull/419/commits/8577280479aebc301087c9cffe32807ba406d352. Fully tested.